### DISCUSSION TITLE
[Enhancement]: Fuctioning "Like" and "Suggest Less' Buttons and implementation via remote

### DISCUSSION THREAD

**Discussion by Ry0men** [ciderapp/Cider#639] [Feature Request]
### The feature you would like

The like button and suggest less doesn’t seem to be working properly, and also if the current playing song on cider shows up on iphone it would be great, so that we can like dislike from there. Thnx and great job in making cider really changed my perspective towards apple music.
Keep it up

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cryptofyre** [comment]
@booploops we might need to use a user token or something for this not sure. AM Web does it using one.
